### PR TITLE
doc: update Error Handle Chinese doc

### PR DESCRIPTION
**Pull request by cqqqq777**
#### What type of PR is this?

docs
#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.
- [ ] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional): update

### CODE REVIEW

**Comment by li-jin-gou** [comment]
thanks

**Comment by li-jin-gou** [comment]
@cqqqq777  没什么问题，可以在最后补充一个代码示例&&英文文档。

File: content/zh/docs/hertz/tutorials/basic-feature/error-handle.md [hunk 1]
```diff
@@ -45,19 +61,85 @@ func New(err error, t ErrorType, meta interface{}) *Error {
...
+
+### 如何使用
+
 对应的 API 为：`RequestContext.Error(err)`，调用该 API 会将 err 绑到对应的请求上下文上之上。
 
-获取请求上下文已绑定的所有错误的方式：`RequestContext.Errors`（ErrorChain）。ErrorChain 目前提供的 API：
+获取请求上下文已绑定的所有错误的方式：`RequestContext.Errors`。
+
+```go
+// 运行此代码并打开游览器访问 localhost:8080/error
```
Comment (li-jin-gou) [inline content/zh/docs/hertz/tutorials/basic-feature/error-handle.md:111]: 地址不对

File: content/zh/docs/hertz/tutorials/basic-feature/error-handle.md [hunk 2]
```diff
@@ -45,19 +61,85 @@ func New(err error, t ErrorType, meta interface{}) *Error {
...
+	_ = c.Error(errors.New("first err"))
+}
+
+func handle2(_ context.Context, c *app.RequestContext) {
+	_ = c.Error(errors.New("second err"))
+}
+
+func handle3(_ context.Context, c *app.RequestContext) {
+	for _, v := range c.Errors.Errors() {
+		c.JSON(consts.StatusOK, v)
```
Comment (li-jin-gou) [inline content/zh/docs/hertz/tutorials/basic-feature/error-handle.md:141]: 额外的循环

**Review by li-jin-gou — APPROVED** [review]

**Comment by li-jin-gou** [comment]
看了一下没啥问题了，大家可以再 review 一下 cc @GuangmingLuo @rogerogers @welkeyever

**Review by rogerogers — DISMISSED** [review]

File: content/en/docs/hertz/tutorials/basic-feature/error-handle.md [hunk 3]
```diff
@@ -1,12 +1,26 @@
 ---
 title: "Error Handle"
-date: 2022-05-23
+date: 2023-04-2
```
Comment (GuangmingLuo) [inline content/en/docs/hertz/tutorials/basic-feature/error-handle.md:3]: 不能写 -04-2，要么 04-02，要么 04-20，和中文保持一致

Comment (li-jin-gou) [reply content/en/docs/hertz/tutorials/basic-feature/error-handle.md]: done

**Review by li-jin-gou — APPROVED** [review]

**Review by rogerogers — APPROVED** [review]

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**